sdk/src/Services/S3/Custom/Transfer/_async/TransferUtility.async.cs

```diff
@@ -230,11 +236,15 @@ public partial class TransferUtility : ITransferUtility
         ///     A cancellation token that can be used by other objects or threads to receive notice of cancellation.
         /// </param>
         /// <returns>The task object representing the asynchronous operation.</returns>
-        public Task AbortMultipartUploadsAsync(string bucketName, DateTime initiatedDate, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task AbortMultipartUploadsAsync(string bucketName, DateTime initiatedDate, CancellationToken cancellationToken = default(CancellationToken))
         {
-            CheckForBlockedArn(bucketName, "AbortMultipartUploads");
-            var command = new AbortMultipartUploadsCommand(this._s3Client, bucketName, initiatedDate, this._config);
-            return command.ExecuteAsync(cancellationToken);
+            using (var span = AWSConfigs.TelemetryProvider.TracerProvider.GetTracer(S3TransferTracerScope)
+                .CreateSpan($"{nameof(TransferUtility)}.{nameof(AbortMultipartUploadsAsync)}", null, Amazon.Runtime.Telemetry.Tracing.SpanKind.CLIENT))
```

Comment:
I would prefer if we don't add `async` to the spans names to be consistent with the SDK (we use `S3.PutObject` for example), but it is up to you.

Reply:
Done